WalletWasabi.Tests/UnitTests/BitcoinCore/IndexBuilderServiceTests.cs

```diff
@@ -32,7 +32,7 @@ public async Task UnsynchronizedBitcoinNodeAsync()
 			indexer.Synchronize();
 
 			await Task.Delay(TimeSpan.FromSeconds(1));
-			// Assert.False(indexer.IsRunning);     // <------------ ERROR: it should have stopped but there is a bug for RegTest
+			//// Assert.False(indexer.IsRunning);     // <------------ ERROR: it should have stopped but there is a bug for RegTest
```

Comment:
Does this make sense?

Reply:
#3613 enabled a StyleCop (CodeFactor) parameter which is `SingleLineCommentMustBePrecededByBlankLine`

So to not have CodeFactor issues we must precede any single line comment by a blank line.
But if the commented out line is a line of code (which is the case with `Assert.False(indexer.IsRunning);`) we can use 4 `/` to comment out that line of code.

For more details take a look at this: https://documentation.help/StyleCop/SA1515.html

So your suggestion would be accepted only if we add a blank line before it.

Reply:
ah, now I understand! 
Then this is good. ACK